overview/default.nix

```diff
@@ -232,19 +232,20 @@ let
     '';
 
     deliverableTags = rec {
-      one = label: ''
-        <span class="deliverable-tag">${label}</span>
+      one = name: label: ''
```

Comment:
Note for later: At this point we should probably write module system type for the output, with an `outPath` or `__toString` option that will display the value. All this processing obscures the structure to the point I can barely follow.

Reply:
Oh yeah that sounds sensible, I think until now I basically followed the pre-existing paradigm and I might have overstretched its usefulness already.